pkgs/development/python-modules/asgi-lifespan/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, sniffio
+, pytestCheckHook
+, starlette
+, trio
+}:
+
+buildPythonPackage rec {
+  pname = "asgi-lifespan";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "florimondmanca";
+    repo = "asgi-lifespan";
+    rev = version;
+    sha256 = "sha256-QooFNxtGQYCRuITvlz3Mkd61XOwnKj6ctBzl1lJkRYE=";
+  };
+
+  postPatch = ''
+    # Remove coverage tests
+    sed -i '21,24d' setup.cfg
```

```suggestion
    sed -i "/--cov/d" setup.cfg
```

Too fragile otherwise.